Sources/Services/AWSS3/Tests/AWSS3Tests/DeleteObjectTaggingRequestTest.swift

```diff
@@ -12,6 +12,7 @@ import ClientRuntime
 import Smithy
 import SmithyTestUtil
 import XCTest
+import func SmithyTestUtil.dummyIdentityResolver
```

Comment:
The 4 tests below demonstrate the code changes to protocol tests.

The `SmithyTestUtil.dummyIdentityResolver` function provides a static identity resolver with some dummy AWS credentials.